### PR TITLE
feat(Day 11): Styling with Tailwind CSS

### PR DESCRIPTION
**Pull request by ruchamahabal**
Improved styling and responsiveness with [Tailwind CSS](https://tailwindcss.com/)

https://user-images.githubusercontent.com/24353136/209527122-7f7c45fb-96a1-4c29-9c98-a5ab022e2e35.mp4

### CODE REVIEW

File: 11_styling_with_tailwind/src/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+					</div>
+				)}
+			</div>
+		</div>
+	);
+};
+
+export default Card;
+
+// .details {
```
Comment (Pujarini) [inline 11_styling_with_tailwind/src/components/Card.js]: Remove commented code. Keep your components as clean as you can

File: 11_styling_with_tailwind/src/components/MemberDetails.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		name,
+		login,
+		followers,
+		following,
+		location,
+		company,
+	} = member;
+
+	return (
+		<div className="member-info items-left flex flex-col justify-center p-24">
```
Comment (Pujarini) [inline 11_styling_with_tailwind/src/components/MemberDetails.js]: why is member-infp present in classname when using tailwind css?

File: 11_styling_with_tailwind/src/components/MemberDetails.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+			<div className="profile flex items-center gap-8 border-b border-solid border-slate-300 pb-14">
+				<img
+					className="member-img member-img h-64 max-w-full rounded-full border-x border-y border-solid border-slate-200 object-cover p-2"
```
Comment (Pujarini) [inline 11_styling_with_tailwind/src/components/MemberDetails.js]: here also i can see member-img

File: 11_styling_with_tailwind/src/components/MemberDetails.js (hunk 2 above)
Comment (Pujarini) [inline 11_styling_with_tailwind/src/components/MemberDetails.js]: try to refactor this code!

Comment (Pujarini) [reply 11_styling_with_tailwind/src/components/MemberDetails.js]: It is too large and little bit messy

**Review by Pujarini — CHANGES_REQUESTED** [review]
reviewed! Please keep your components as small as you can and try to refactor

File: 11_styling_with_tailwind/src/components/Card.js (hunk 1 above)
Comment (ruchamahabal) [reply 11_styling_with_tailwind/src/components/Card.js]: Yeah, I kept it for reference while trying to convert styles to classes. Forgot to remove it. Thanks for pointing out :P

File: 11_styling_with_tailwind/src/components/MemberDetails.js (hunk 2 above)
Comment (ruchamahabal) [reply 11_styling_with_tailwind/src/components/MemberDetails.js]: These were older class names where styles were applied. Removed

**Pull request merged**